docs/application/native/guides/multimedia/sound-manager.md

```diff
@@ -181,6 +181,53 @@ To query sound device information:
           /* Failed to free the device list*/
       ```
 
+   5. More functionality for `SOUND_DEVICE_USB_AUDIO` with `SOUND_DEVICE_IO_DIRECTION_OUT` device.
+      When you found a device with above type, you can utilize more functionality with the following functions:
+
+      - `sound_manager_get_supported_sample_formats()`: To get the supported sample formats of the device.
+      - `sound_manager_set_sample_format()`: To set the sample format to the device.
+      - `sound_manager_get_sample_format()`: To get the sample format of the device.
+      - `sound_manager_get_supported_sample_rates()`: To get the supported sample rates of the device.
+      - `sound_manager_set_sample_rate()`: To set the sample rate to the device.
+      - `sound_manager_get_sample_rate()`: To get the sample rate of the device.
+      - `sound_manager_set_media_stream_only()`: To set the 'media stream only' property. With this enabled, no other stream type except `SOUND_STREAM_TYPE_MEDIA` is not allowed to this device.
+      - `sound_manager_get_media_stream_only()`: To get the 'media stream only' property.
+      - `sound_manager_set_avoid_resampling()`: To set the 'avoid resampling' property. With this enabled, this device will use the first stream's original sample format and rate without resampling if supported.
```

Comment:
A suggestion to avoid using apostrophe " 's ". Kindly re-phrase the sentence like: 

To set the 'avoid resampling' property. With this enabled, the device uses the original sample format of the first stream and the sample rate without resampling (if supported).

Please verify the technical information on this as the sentence is not clear to me?

Reply:
I'd like to change it like this, is it fine to you?

With this enabled, the device uses the original sample format and rate of the first stream without resampling if supported by the device.